File: AngularMicroservice/src/dto/ActionEventDTO.ts
import { LabelDTO } from './LabelDTO';

export class ActioneventDTO{
    id: number;
    name: String;
    description: String;
    type: number;
 

    constructor(id: number,  description: String, name: String, type:number){
        this.id = id;
        this.description= description;
        this.name= name;

        this.type= type;

    }
    
}

File: RobotMicroservice/src/main/java/it/contrader/dto/ActionEventDTO.java
package it.contrader.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.ToString;

@Data
@AllArgsConstructor
@NoArgsConstructor
@ToString
public class ActionEventDTO {
	
	private Integer idActionEvent;
	private String description;
	private String name;
	private LabelDTO label;
	private Integer type;

}